c2cgeoportal/scripts/c2ctool.py

```diff
@@ -119,6 +119,12 @@ def _fill_arguments(command):
         "--windows",
         help="Use the windows c2cgeoportal package",
     )
+    parser.add_argument(
+        "--gitrepository",
+        metavar="origin",
```

Comment:
Why do you set the metavar ?
The default value is GITREPOSITORY and it should be something like this...